TCAT/Views/WhatsNewHeaderView.swift

```diff
@@ -37,36 +37,35 @@ class WhatsNewHeaderView: UIView {
         createUpdateTitle(updateName: updateName)
         createDismissButton()
     }
-    
+
     func createWhatsNewHeader() {
         whatsNewHeader = UILabel()
         whatsNewHeader.text = "NEW IN ITHACA TRANSIT"
-        whatsNewHeader.font = UIFont(name: Constants.Fonts.SanFrancisco.Semibold, size: 10)
+        whatsNewHeader.font = UIFont(name: Constants.Fonts.SanFrancisco.Semibold, size: 12)
```

Comment:
Please merge with master and use `UIFont.style(Fonts.SanFrancisco.semibold, size: 12)`. I won't comment every other line, but Command-F any time you added `UIFont(`